scripts/launch-s0-instance.sh

```diff
@@ -3,81 +3,233 @@
+      Restart=always
+      RestartSec=5
+
+      [Install]
+      WantedBy=multi-user.target
+
+runcmd:
+  # Format and mount nvme disk
+  - |
+    mkdir -p $DB_MOUNT
+    if [[ -b /dev/nvme1n1 && -b /dev/nvme2n1 ]]; then
+      # Two disks; make an array
+      mdadm --create --verbose /dev/md0 --level=0 --raid-devices=2 /dev/nvme1n1 /dev/nvme2n1
+      mkfs.xfs -f /dev/md0
+      mount /dev/md0 $DB_MOUNT
+    elif [ -b /dev/nvme1n1 ]; then
+      # One disk; no need to involve mdadm
+      parted -a optimal --script /dev/nvme1n1 mklabel gpt
+      parted -a optimal --script /dev/nvme1n1 mkpart primary 0% 100%
+      mkfs.xfs -f /dev/nvme1n1p1
+      mount /dev/nvme1n1p1 $DB_MOUNT
+    fi
+    chown -R ec2-user:ec2-user $DB_MOUNT
+
+  # Configure AWS region
+  - |
+    echo "region = \$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep -w region | cut -d: -f2 | tr -d '", ')" >> /home/ec2-user/.aws/config
+    cp -r /home/ec2-user/.aws /root/
+
+  # Create s0 vault
+  - |
+    s0 vault create s3 --bucket $S0_BUCKET --key-id $S0_KMS 2>&1 | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g" > /var/log/s0-vault-create.log
+
+  # Prepare and start s0 service
+  - |
+    touch /var/log/s0-service.log
+    chown ec2-user:ec2-user /var/log/s0-service.log
+    systemctl daemon-reload
+    systemctl enable $UNIT_FILE
+    systemctl start $SERVICE
+
+output:
+  all: '| tee -a /var/log/cloud-init-output.log'
+
+EOF
+}
 
 usage()
 {
     echo "Usage examples:"
-    echo "   $me --key-name my_ec2_key --name my_s0_server --type m5d.2xlarge"
-    echo "   $me --k /path/to/key.pem -n my_s0_server -t m5d.2xlarge"
+    echo "   $me --ssh-key-name my_ec2_key --instance-name my_s0_server --instance-type m5d.2xlarge --kms-key-alias my_kms_key --bucket-name my_s3_bucket"
+    echo "   $me -s my_ec2_key -n my_s0_server -t m5ad.2xlarge -k my_kms_key -b my_s3_bucket"
+    echo
+    echo "  -s | --ssh-key-name       : A name of an aws EC2 key in the PEM format for the SSH connection to the instance."
+    echo
+    echo "  -n | --instance-name      : Optional. Name of an instance. It's shown in the AWS console."
+    echo "                              The default value is \"[user_name]-s0-server\"."
+    echo
+    echo "  -t | --instance-type      : Optional. An instance type. See https://aws.amazon.com/ec2/instance-types/ for more details."
+    echo "                              The default value is \"m5d.2xlarge\"."
+    echo
+    echo "  -g | --security-group     : Optional. A name of the security group which has ports 22 and 61234 open to the world for SSH and s0 server respectively."
+    echo "                              The grop 'elastio-s0-server' will be created and then used in case if this OR next argument is not specified."
```

Comment:
```suggestion
    echo "                              The group 'elastio-s0-server' will be created and then used in case if this OR next argument is not specified."
```